src/test/kotlin/com/linchpino/core/controller/AuthenticationControllerTestIT.kt

```diff
@@ -0,0 +1,77 @@
+package com.linchpino.core.controller
+
+import com.linchpino.core.PostgresContainerConfig
+import com.linchpino.core.dto.CreateAccountRequest
+import com.linchpino.core.enums.AccountStatusEnum
+import com.linchpino.core.repository.AccountRepository
+import com.linchpino.core.service.AccountService
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.httpBasic
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import org.springframework.transaction.annotation.Transactional
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@Transactional // Ensure rollback after each test
+@Import(PostgresContainerConfig::class)
+class AuthenticationControllerTestIT {
```

Comment:
Please add two more cases:
1. A user with an inactive status can not log in
2. The email address can be Case Insitive